.github/workflows/tests.yml

```
@@ -0,0 +1,36 @@
+name: Tests
```

```suggestion
name: Latest Version Notebook Tests
```

to match `test_minimum_versions.yml`.